spec/models/concerns/type_of_means_assessment_spec.rb

```diff
@@ -514,6 +514,7 @@
     end
 
     context 'defaults to applicant' do
+      let(:partner) { instance_double(Partner) }
```

Comment:
Fixes rspec warning: `WARNING: An expectation of `:benefit_type` was set on `nil`. To allow expectations on `nil` and suppress this message, set `RSpec::Mocks.configuration.allow_message_expectations_on_nil` to `true`. To disallow expectations on `nil`, set `RSpec::Mocks.configuration.allow_message_expectations_on_nil` to `false`. Called from /Users/hibo.abdilaahi/Documents/GitHub/LAA/laa-apply-for-criminal-legal-aid/spec/models/concerns/type_of_means_assessment_spec.rb:523:in `block (4 levels) in <top (required)>'.`